providers/coordinator.go

```diff
@@ -197,8 +198,7 @@ func (c *coordinator) Start(id string, isEphemeral bool, update UpdateProvidersC
 		}
 	}
 
-	pluginCmd := exec.Command(provider.binPath(), "run_as_plugin")
-	log.Debug().Str("path", pluginCmd.Path).Msg("running provider plugin")
+	pluginCmd := exec.Command(provider.binPath(), []string{"--run-as-plugin", "--log-level", zerolog.GlobalLevel().String()}...)
```

Comment:
This would not be backwards-compatible with older plugin versions. Any reason why not keep the subcommand "run_as_plugin"?

Reply:
I didn't read it as subcommand, but thought it was another arg. Reverted this part.